packages/components/src/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -395,6 +411,13 @@ class SampleTypeDesignerImpl extends React.PureComponent<Props & InjectedBaseDom
                     helpTopic={helpTopic}
                     model={model}
                     parentOptions={parentOptions}
+                    includeDataClasses={includeDataClasses}
+                    useSeparateDataClasses={useSeparateDataClasses}
+                    sampleAliasCaption={sampleAliasCaption}
+                    sampleTypeCaption={sampleTypeCaption}
+                    dataClassAliasCaption={dataClassAliasCaption}
+                    dataClassTypeCaption={dataClassTypeCaption}
+                    dataClassParentageLabel={dataClassParentageLabel}
```

Comment:
Since we're passing all of these properties around as a unit, I wonder if it would be better to collect them into a single property and define an interface.

Reply:
I like this idea, but will probably leave it as is to allow different combinations of knobs for storybook.